distributed/center.py

```diff
@@ -108,21 +109,23 @@ def port(self):
     @gen.coroutine
     def terminate(self, stream=None):
         self.stop()
-        return b'OK'
+        return 'OK'
 
     def register(self, stream, address=None, keys=(), ncores=None,
                  services=None):
+        address = coerce_to_address(address)
         self.has_what[address] = set(keys)
         for key in keys:
             self.who_has[key].add(address)
         self.ncores[address] = ncores
         self.worker_services[address] = services
         logger.info("Register %s", str(address))
-        return b'OK'
+        return 'OK'
 
     def unregister(self, stream, address=None):
+        address = coerce_to_address(address)
         if address not in self.has_what:
-            return b'Address not found: ' + str(address).encode()
+            return 'Address not found: ' + str(address).encode()
```

Comment:
It should actually just be `address`.  In the zeal of refactoring I ended up crippling my testing framework.  Working through this and similar issues that snuck by now.